packages/scripts/src/lib/app-data/postProcess/assets.ts

```diff
@@ -34,27 +35,32 @@ const APP_CORE_SIZE_KB = {
   total: APP_CORE_BUILD_KB + APP_CORE_ASSETS_KB,
 };
 
+interface IAssetFolderMetadata {
+  /** Whether this folder contains remote assets (asset pack) */
+  remote?: boolean;
+  /** Name of the folder (used for remote asset pack output directory) */
+  folderName?: string;
+}
+
 interface IAssetPostProcessorOptions {
   sourceAssetsFolders: string[];
+  /** Map of source folder path to metadata (remote flag and folder name) */
+  folderMetadata?: Map<string, IAssetFolderMetadata>;
```

Comment:
nit(blocking): I think the list of folders is the same as the keys for the metadata no? If so then let's just pass a single list to iterate over.

I'd also probably recommend sticking with arrays over maps in case we ever need funcationality that merges multiple input sources into a single remote asset pack (otherwise remote folder names would override each other)